src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -211,6 +211,14 @@ export default class AddStaffNotationDialog extends Vue {
     return ''
   }
 
+  get notationLabel (): string {
+    if (this.isCourtOrder) {
+      return `${this.displayName} Text`
+    } else {
+      return `${this.displayName}`
+    }
+  }
```

Comment:
What is the UX requirement that led to this change. Can it be a common change for all staff notation filings?

Reply:
![image](https://user-images.githubusercontent.com/116035339/213548530-c5daebc8-79b7-46bb-92cf-7e3c7d8338e8.png)

Reply:
Beautiful 👏